demo-projects/blog/package.json

```diff
@@ -33,14 +32,11 @@
     "@keystonejs/file-adapters": "^7.0.2",
     "@keystonejs/keystone": "^12.0.0",
     "@keystonejs/oembed-adapters": "^5.1.4",
-    "apollo-cache-inmemory": "^1.6.6",
-    "apollo-client": "^2.6.10",
-    "apollo-upload-client": "^13.0.0",
+    "apollo-upload-client": "^14.1.0",
     "cross-env": "^7.0.0",
     "date-fns": "^2.14.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "graphql-tag": "^2.10.3",
     "isomorphic-unfetch": "^3.0.0",
```

Comment:
I think we should add `graphql` as a dependency here as per the [apollo docs](https://www.apollographql.com/docs/react/get-started/). It looks like its just missing from the demo projects

Reply:
Done

Reply:
Thanks @Vultraz !